test/build/dataflow/test_depscheck.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import unittest
+
+from unittest import mock
+
+from test import mixins
+
+import bigflow.build.pip as bf_pip
+import bigflow.build.dataflow.depscheck as dc
+
+
+class DataflowDepsCheckTestCase(
+    mixins.TempCwdMixin,
+    mixins.FileUtilsMixin,
+    unittest.TestCase,
+):
+    def setUp(self):
```

Comment:
It would be nice to have a E2E based on the CLI and a real project, what do you think?

Reply:
IMO it doesn't make a big sense - it interacts only with `pip-compile` tool.
And rest of `bigflow` knows nothing about such pinning (including `bigflow.dataflow` package).

Reply:
ok